### PR TITLE
chore(vault-secrets-manager): create lib for vault-secrets-manager

### PR DESCRIPTION
**Pull request by smark88**
This is to replace https://github.com/influxdata/k8s-infra/blob/master/lib/vaultSecrets.libsonnet with a more complete lib

### CODE REVIEW

**Review by primeroz — APPROVED** [review]

File: gen/github.com/jsonnet-libs/vault-secrets-manager-libsonnet/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
[the remark this answers is not in the document]
Comment (smark88) [reply gen/github.com/jsonnet-libs/vault-secrets-manager-libsonnet/LICENSE:1]: auto generated

**Comment by primeroz** [comment]
careful that i doubt our version will match this new upstream one 100%

**Comment by smark88** [comment]
> careful that i doubt our version will match this new upstream one 100%

It should be the latest release

**Comment by primeroz** [comment]
the one we use now i wrote it by hand

**Pull request merged**